diff --git a/vendor/netbox-client/src/netbox-client/models/brief_module.cr b/vendor/netbox-client/src/netbox-client/models/brief_module.cr
--- a/vendor/netbox-client/src/netbox-client/models/brief_module.cr
+++ b/vendor/netbox-client/src/netbox-client/models/brief_module.cr
@@ -25,10 +25,10 @@
     property display : String
 
     @[JSON::Field(key: "device", emit_null: false)]
-    property device : BriefDevice
+    property device : BriefDevice?
 
     @[JSON::Field(key: "module_bay", emit_null: false)]
-    property module_bay : NestedModuleBay
+    property module_bay : NestedModuleBay?
 
 
     # Initializes the object
