diff --git a/vendor/netbox-client/src/netbox-client/models/ip_address.cr b/vendor/netbox-client/src/netbox-client/models/ip_address.cr
--- a/vendor/netbox-client/src/netbox-client/models/ip_address.cr
+++ b/vendor/netbox-client/src/netbox-client/models/ip_address.cr
@@ -34,7 +34,7 @@
     property address : String
 
     @[JSON::Field(key: "assigned_object", emit_null: false)]
-    property assigned_object : JSON::Any
+    property assigned_object : JSON::Any?
 
     @[JSON::Field(key: "nat_outside", emit_null: false)]
     property nat_outside : Array(NestedIPAddress)
