diff --git a/vendor/netbox-client/src/netbox-client/models/object_permission.cr b/vendor/netbox-client/src/netbox-client/models/object_permission.cr
--- a/vendor/netbox-client/src/netbox-client/models/object_permission.cr
+++ b/vendor/netbox-client/src/netbox-client/models/object_permission.cr
@@ -22,7 +22,7 @@
     property url : String
 
     @[JSON::Field(key: "display_url", emit_null: false)]
-    property display_url : String
+    property display_url : String?
 
     @[JSON::Field(key: "display", emit_null: false)]
     property display : String
