diff --git a/vendor/netbox-client/src/netbox-client/models/virtual_machine_with_config_context.cr b/vendor/netbox-client/src/netbox-client/models/virtual_machine_with_config_context.cr
--- a/vendor/netbox-client/src/netbox-client/models/virtual_machine_with_config_context.cr
+++ b/vendor/netbox-client/src/netbox-client/models/virtual_machine_with_config_context.cr
@@ -31,10 +31,10 @@
     property name : String
 
     @[JSON::Field(key: "primary_ip", emit_null: false)]
-    property primary_ip : BriefIPAddress
+    property primary_ip : BriefIPAddress?
 
     @[JSON::Field(key: "config_context", emit_null: false)]
-    property config_context : JSON::Any
+    property config_context : JSON::Any?
 
     @[JSON::Field(key: "created", emit_null: false)]
     property created : Time
