diff --git a/vendor/netbox-client/src/netbox-client/models/brief_rack.cr b/vendor/netbox-client/src/netbox-client/models/brief_rack.cr
--- a/vendor/netbox-client/src/netbox-client/models/brief_rack.cr
+++ b/vendor/netbox-client/src/netbox-client/models/brief_rack.cr
@@ -28,7 +28,7 @@
     property name : String
 
     @[JSON::Field(key: "device_count", emit_null: false)]
-    property device_count : Int64
+    property device_count : Int64?
 
     # Optional properties
     @[JSON::Field(key: "description", emit_null: false)]
