diff --git a/vendor/netbox-client/src/netbox-client/models/prefix.cr b/vendor/netbox-client/src/netbox-client/models/prefix.cr
--- a/vendor/netbox-client/src/netbox-client/models/prefix.cr
+++ b/vendor/netbox-client/src/netbox-client/models/prefix.cr
@@ -34,7 +34,7 @@
     property prefix : String
 
     @[JSON::Field(key: "scope", emit_null: false)]
-    property scope : JSON::Any
+    property scope : JSON::Any?
 
     @[JSON::Field(key: "created", emit_null: false)]
     property created : Time
