diff --git a/vendor/netbox-client/src/netbox-client/models/cluster.cr b/vendor/netbox-client/src/netbox-client/models/cluster.cr
--- a/vendor/netbox-client/src/netbox-client/models/cluster.cr
+++ b/vendor/netbox-client/src/netbox-client/models/cluster.cr
@@ -34,7 +34,7 @@
     property _type : BriefClusterType
 
     @[JSON::Field(key: "scope", emit_null: false)]
-    property scope : JSON::Any
+    property scope : JSON::Any?
 
     @[JSON::Field(key: "created", emit_null: false)]
     property created : Time
@@ -49,13 +49,13 @@
     property virtualmachine_count : Int64
 
     @[JSON::Field(key: "allocated_vcpus", emit_null: false)]
-    property allocated_vcpus : Float64
+    property allocated_vcpus : Float64?
 
     @[JSON::Field(key: "allocated_memory", emit_null: false)]
-    property allocated_memory : Int32
+    property allocated_memory : Int32?
 
     @[JSON::Field(key: "allocated_disk", emit_null: false)]
-    property allocated_disk : Int32
+    property allocated_disk : Int32?
 
     # Optional properties
     @[JSON::Field(key: "group", emit_null: false)]
