diff --git a/vendor/netbox-client/src/netbox-client/models/brief_circuit_type.cr b/vendor/netbox-client/src/netbox-client/models/brief_circuit_type.cr
index c6a6047..05a801c 100644
--- a/vendor/netbox-client/src/netbox-client/models/brief_circuit_type.cr
+++ b/vendor/netbox-client/src/netbox-client/models/brief_circuit_type.cr
@@ -35,8 +35,8 @@ module NetboxClient
     @[JSON::Field(key: "slug", type: String, nillable: false, emit_null: false)]
     property slug : String

-    @[JSON::Field(key: "circuit_count", type: Int64, nillable: false, emit_null: false)]
-    property circuit_count : Int64
+    @[JSON::Field(key: "circuit_count", type: Int64?, nillable: true, emit_null: false)]
+    property circuit_count : Int64?

     # Optional properties
     @[JSON::Field(key: "description", type: String?, nillable: true, emit_null: false)]
