diff --git a/vendor/netbox-client/src/netbox-client/models/mac_address.cr b/vendor/netbox-client/src/netbox-client/models/mac_address.cr
--- a/vendor/netbox-client/src/netbox-client/models/mac_address.cr
+++ b/vendor/netbox-client/src/netbox-client/models/mac_address.cr
@@ -31,7 +31,7 @@
     property mac_address : String
 
     @[JSON::Field(key: "assigned_object", emit_null: false)]
-    property assigned_object : JSON::Any
+    property assigned_object : JSON::Any?
 
     @[JSON::Field(key: "created", emit_null: false)]
     property created : Time
