diff --git a/vendor/netbox-client/src/netbox-client/models/brief_cluster.cr b/vendor/netbox-client/src/netbox-client/models/brief_cluster.cr
index cf5b5fa..d066af7 100644
--- a/vendor/netbox-client/src/netbox-client/models/brief_cluster.cr
+++ b/vendor/netbox-client/src/netbox-client/models/brief_cluster.cr
@@ -32,8 +32,8 @@ module NetboxClient
     @[JSON::Field(key: "name", type: String, nillable: false, emit_null: false)]
     property name : String

-    @[JSON::Field(key: "virtualmachine_count", type: Int64, nillable: false, emit_null: false)]
-    property virtualmachine_count : Int64
+    @[JSON::Field(key: "virtualmachine_count", type: Int64?, nillable: true, emit_null: false)]
+    property virtualmachine_count : Int64?

     # Optional properties
     @[JSON::Field(key: "description", type: String?, nillable: true, emit_null: false)]
