diff --git a/vendor/netbox-client/src/netbox-client/models/brief_power_port.cr b/vendor/netbox-client/src/netbox-client/models/brief_power_port.cr
--- a/vendor/netbox-client/src/netbox-client/models/brief_power_port.cr
+++ b/vendor/netbox-client/src/netbox-client/models/brief_power_port.cr
@@ -31,7 +31,7 @@
     property name : String
 
     @[JSON::Field(key: "cable", emit_null: false)]
-    property cable : BriefCable
+    property cable : BriefCable?
 
     @[JSON::Field(key: "_occupied", emit_null: false)]
     property _occupied : Bool
