diff --git a/vendor/netbox-client/src/netbox-client/models/group.cr b/vendor/netbox-client/src/netbox-client/models/group.cr
--- a/vendor/netbox-client/src/netbox-client/models/group.cr
+++ b/vendor/netbox-client/src/netbox-client/models/group.cr
@@ -22,7 +22,7 @@
     property url : String
 
     @[JSON::Field(key: "display_url", emit_null: false)]
-    property display_url : String
+    property display_url : String?
 
     @[JSON::Field(key: "display", emit_null: false)]
     property display : String
@@ -31,7 +31,7 @@
     property name : String
 
     @[JSON::Field(key: "user_count", emit_null: false)]
-    property user_count : Int32
+    property user_count : Int32?
 
     # Optional properties
     @[JSON::Field(key: "description", emit_null: false)]
