diff --git a/vendor/netbox-client/src/netbox-client/models/brief_circuit_type.cr b/vendor/netbox-client/src/netbox-client/models/brief_circuit_type.cr
--- a/vendor/netbox-client/src/netbox-client/models/brief_circuit_type.cr
+++ b/vendor/netbox-client/src/netbox-client/models/brief_circuit_type.cr
@@ -31,7 +31,7 @@
     property slug : String
 
     @[JSON::Field(key: "circuit_count", emit_null: false)]
-    property circuit_count : Int64
+    property circuit_count : Int64?
 
     # Optional properties
     @[JSON::Field(key: "description", emit_null: false)]
