diff --git a/vendor/netbox-client/src/netbox-client/models/brief_role.cr b/vendor/netbox-client/src/netbox-client/models/brief_role.cr
--- a/vendor/netbox-client/src/netbox-client/models/brief_role.cr
+++ b/vendor/netbox-client/src/netbox-client/models/brief_role.cr
@@ -31,10 +31,10 @@
     property slug : String
 
     @[JSON::Field(key: "prefix_count", emit_null: false)]
-    property prefix_count : Int64
+    property prefix_count : Int64?
 
     @[JSON::Field(key: "vlan_count", emit_null: false)]
-    property vlan_count : Int64
+    property vlan_count : Int64?
 
     # Optional properties
     @[JSON::Field(key: "description", emit_null: false)]
