diff --git a/vendor/netbox-client/src/netbox-client/models/job.cr b/vendor/netbox-client/src/netbox-client/models/job.cr
--- a/vendor/netbox-client/src/netbox-client/models/job.cr
+++ b/vendor/netbox-client/src/netbox-client/models/job.cr
@@ -27,10 +27,10 @@
     property display : String
 
     @[JSON::Field(key: "object_type", emit_null: false)]
-    property object_type : String
+    property object_type : String?
 
     @[JSON::Field(key: "object", emit_null: false)]
-    property object : JSON::Any
+    property object : JSON::Any?
 
     @[JSON::Field(key: "name", emit_null: false)]
     property name : String
@@ -42,7 +42,7 @@
     property created : Time
 
     @[JSON::Field(key: "user", emit_null: false)]
-    property user : BriefUser
+    property user : BriefUser?
 
     @[JSON::Field(key: "error", emit_null: false)]
     property error : String
