diff --git a/vendor/netbox-client/src/netbox-client/models/brief_device_type.cr b/vendor/netbox-client/src/netbox-client/models/brief_device_type.cr
--- a/vendor/netbox-client/src/netbox-client/models/brief_device_type.cr
+++ b/vendor/netbox-client/src/netbox-client/models/brief_device_type.cr
@@ -34,7 +34,7 @@
     property slug : String
 
     @[JSON::Field(key: "device_count", emit_null: false)]
-    property device_count : Int64
+    property device_count : Int64?
 
     # Optional properties
     @[JSON::Field(key: "description", emit_null: false)]
