diff --git a/vendor/netbox-client/src/netbox-client/models/object_type.cr b/vendor/netbox-client/src/netbox-client/models/object_type.cr
--- a/vendor/netbox-client/src/netbox-client/models/object_type.cr
+++ b/vendor/netbox-client/src/netbox-client/models/object_type.cr
@@ -48,7 +48,7 @@
     property is_plugin_model : Bool
 
     @[JSON::Field(key: "rest_api_endpoint", emit_null: false)]
-    property rest_api_endpoint : String
+    property rest_api_endpoint : String?
 
     @[JSON::Field(key: "description", emit_null: false)]
     property description : String
