diff --git a/vendor/netbox-client/src/netbox-client/models/brief_rack_role.cr b/vendor/netbox-client/src/netbox-client/models/brief_rack_role.cr
--- a/vendor/netbox-client/src/netbox-client/models/brief_rack_role.cr
+++ b/vendor/netbox-client/src/netbox-client/models/brief_rack_role.cr
@@ -31,7 +31,7 @@
     property slug : String
 
     @[JSON::Field(key: "rack_count", emit_null: false)]
-    property rack_count : Int64
+    property rack_count : Int64?
 
     # Optional properties
     @[JSON::Field(key: "description", emit_null: false)]
