diff --git a/vendor/netbox-client/src/netbox-client/models/brief_cluster_group.cr b/vendor/netbox-client/src/netbox-client/models/brief_cluster_group.cr
--- a/vendor/netbox-client/src/netbox-client/models/brief_cluster_group.cr
+++ b/vendor/netbox-client/src/netbox-client/models/brief_cluster_group.cr
@@ -31,7 +31,7 @@
     property slug : String
 
     @[JSON::Field(key: "cluster_count", emit_null: false)]
-    property cluster_count : Int64
+    property cluster_count : Int64?
 
     # Optional properties
     @[JSON::Field(key: "description", emit_null: false)]
