diff --git a/vendor/netbox-client/src/netbox-client/models/console_port.cr b/vendor/netbox-client/src/netbox-client/models/console_port.cr
--- a/vendor/netbox-client/src/netbox-client/models/console_port.cr
+++ b/vendor/netbox-client/src/netbox-client/models/console_port.cr
@@ -34,26 +34,26 @@
     property name : String
 
     @[JSON::Field(key: "cable", emit_null: false)]
-    property cable : BriefCable
+    property cable : BriefCable?
 
     @[JSON::Field(key: "cable_end", emit_null: false)]
-    property cable_end : String
+    property cable_end : String?
 
     @[JSON::Field(key: "link_peers", emit_null: false)]
     property link_peers : Array(JSON::Any)
 
     # Return the type of the peer link terminations, or None.
     @[JSON::Field(key: "link_peers_type", emit_null: false)]
-    property link_peers_type : String
+    property link_peers_type : String?
 
     @[JSON::Field(key: "connected_endpoints", emit_null: false)]
-    property connected_endpoints : Array(JSON::Any)
+    property connected_endpoints : Array(JSON::Any)?
 
     @[JSON::Field(key: "connected_endpoints_type", emit_null: false)]
-    property connected_endpoints_type : String
+    property connected_endpoints_type : String?
 
     @[JSON::Field(key: "connected_endpoints_reachable", emit_null: false)]
-    property connected_endpoints_reachable : Bool
+    property connected_endpoints_reachable : Bool?
 
     @[JSON::Field(key: "created", emit_null: false)]
     property created : Time
