diff --git a/vendor/netbox-client/src/netbox-client/models/available_ip.cr b/vendor/netbox-client/src/netbox-client/models/available_ip.cr
--- a/vendor/netbox-client/src/netbox-client/models/available_ip.cr
+++ b/vendor/netbox-client/src/netbox-client/models/available_ip.cr
@@ -22,7 +22,7 @@
     property address : String
 
     @[JSON::Field(key: "vrf", emit_null: false)]
-    property vrf : BriefVRF
+    property vrf : BriefVRF?
 
     # Optional properties
     @[JSON::Field(key: "description", emit_null: false)]
