diff --git a/vendor/netbox-client/src/netbox-client/models/vm_interface.cr b/vendor/netbox-client/src/netbox-client/models/vm_interface.cr
--- a/vendor/netbox-client/src/netbox-client/models/vm_interface.cr
+++ b/vendor/netbox-client/src/netbox-client/models/vm_interface.cr
@@ -34,13 +34,13 @@
     property name : String
 
     @[JSON::Field(key: "mac_address", emit_null: false)]
-    property mac_address : String
+    property mac_address : String?
 
     @[JSON::Field(key: "mac_addresses", emit_null: false)]
     property mac_addresses : Array(BriefMACAddress)
 
     @[JSON::Field(key: "l2vpn_termination", emit_null: false)]
-    property l2vpn_termination : BriefL2VPNTermination
+    property l2vpn_termination : BriefL2VPNTermination?
 
     @[JSON::Field(key: "created", emit_null: false)]
     property created : Time
