diff --git a/vendor/netbox-client/src/netbox-client/models/brief_rir.cr b/vendor/netbox-client/src/netbox-client/models/brief_rir.cr
index bca2f2e..d4afccf 100644
--- a/vendor/netbox-client/src/netbox-client/models/brief_rir.cr
+++ b/vendor/netbox-client/src/netbox-client/models/brief_rir.cr
@@ -35,8 +35,8 @@ module NetboxClient
     @[JSON::Field(key: "slug", type: String, nillable: false, emit_null: false)]
     property slug : String

-    @[JSON::Field(key: "aggregate_count", type: Int64, nillable: false, emit_null: false)]
-    property aggregate_count : Int64
+    @[JSON::Field(key: "aggregate_count", type: Int64?, nillable: true, emit_null: false)]
+    property aggregate_count : Int64?

     # Optional properties
     @[JSON::Field(key: "description", type: String?, nillable: true, emit_null: false)]
