diff --git a/vendor/netbox-client/src/netbox-client/models/circuit.cr b/vendor/netbox-client/src/netbox-client/models/circuit.cr
--- a/vendor/netbox-client/src/netbox-client/models/circuit.cr
+++ b/vendor/netbox-client/src/netbox-client/models/circuit.cr
@@ -38,10 +38,10 @@
     property _type : BriefCircuitType
 
     @[JSON::Field(key: "termination_a", emit_null: false)]
-    property termination_a : CircuitCircuitTermination
+    property termination_a : CircuitCircuitTermination?
 
     @[JSON::Field(key: "termination_z", emit_null: false)]
-    property termination_z : CircuitCircuitTermination
+    property termination_z : CircuitCircuitTermination?
 
     @[JSON::Field(key: "created", emit_null: false)]
     property created : Time
