diff --git a/vendor/netbox-client/src/netbox-client/models/brief_inventory_item_role.cr b/vendor/netbox-client/src/netbox-client/models/brief_inventory_item_role.cr
--- a/vendor/netbox-client/src/netbox-client/models/brief_inventory_item_role.cr
+++ b/vendor/netbox-client/src/netbox-client/models/brief_inventory_item_role.cr
@@ -31,7 +31,7 @@
     property slug : String
 
     @[JSON::Field(key: "inventoryitem_count", emit_null: false)]
-    property inventoryitem_count : Int64
+    property inventoryitem_count : Int64?
 
     # Optional properties
     @[JSON::Field(key: "description", emit_null: false)]
