diff --git a/vendor/netbox-client/src/netbox-client/models/front_port.cr b/vendor/netbox-client/src/netbox-client/models/front_port.cr
--- a/vendor/netbox-client/src/netbox-client/models/front_port.cr
+++ b/vendor/netbox-client/src/netbox-client/models/front_port.cr
@@ -40,17 +40,17 @@
     property rear_port : FrontPortRearPort
 
     @[JSON::Field(key: "cable", emit_null: false)]
-    property cable : BriefCable
+    property cable : BriefCable?
 
     @[JSON::Field(key: "cable_end", emit_null: false)]
-    property cable_end : String
+    property cable_end : String?
 
     @[JSON::Field(key: "link_peers", emit_null: false)]
     property link_peers : Array(JSON::Any)
 
     # Return the type of the peer link terminations, or None.
     @[JSON::Field(key: "link_peers_type", emit_null: false)]
-    property link_peers_type : String
+    property link_peers_type : String?
 
     @[JSON::Field(key: "created", emit_null: false)]
     property created : Time
