diff --git a/vendor/netbox-client/src/netbox-client/models/object_change.cr b/vendor/netbox-client/src/netbox-client/models/object_change.cr
--- a/vendor/netbox-client/src/netbox-client/models/object_change.cr
+++ b/vendor/netbox-client/src/netbox-client/models/object_change.cr
@@ -57,7 +57,7 @@
     property prechange_data : JSON::Any
 
     @[JSON::Field(key: "postchange_data", emit_null: false)]
-    property postchange_data : JSON::Any
+    property postchange_data : JSON::Any?
 
     validates(changed_object_id, Int64, false, maximum: 9223372036854775807, minimum: 0)
 
