diff --git a/vendor/netbox-client/src/netbox-client/models/brief_vrf.cr b/vendor/netbox-client/src/netbox-client/models/brief_vrf.cr
--- a/vendor/netbox-client/src/netbox-client/models/brief_vrf.cr
+++ b/vendor/netbox-client/src/netbox-client/models/brief_vrf.cr
@@ -28,7 +28,7 @@
     property name : String
 
     @[JSON::Field(key: "prefix_count", emit_null: false)]
-    property prefix_count : Int64
+    property prefix_count : Int64?
 
     # Optional properties
     # Unique route distinguisher (as defined in RFC 4364)
