diff --git a/vendor/netbox-client/src/netbox-client/models/config_context.cr b/vendor/netbox-client/src/netbox-client/models/config_context.cr
--- a/vendor/netbox-client/src/netbox-client/models/config_context.cr
+++ b/vendor/netbox-client/src/netbox-client/models/config_context.cr
@@ -35,10 +35,10 @@
     property data_path : String
 
     @[JSON::Field(key: "data_file", emit_null: false)]
-    property data_file : BriefDataFile
+    property data_file : BriefDataFile?
 
     @[JSON::Field(key: "data_synced", emit_null: false)]
-    property data_synced : Time
+    property data_synced : Time?
 
     @[JSON::Field(key: "data", emit_null: false)]
     property data : JSON::Any
