diff --git a/vendor/netbox-client/src/netbox-client/models/available_prefix.cr b/vendor/netbox-client/src/netbox-client/models/available_prefix.cr
--- a/vendor/netbox-client/src/netbox-client/models/available_prefix.cr
+++ b/vendor/netbox-client/src/netbox-client/models/available_prefix.cr
@@ -22,7 +22,7 @@
     property prefix : String
 
     @[JSON::Field(key: "vrf", emit_null: false)]
-    property vrf : BriefVRF
+    property vrf : BriefVRF?
 
 
     # Initializes the object
