diff --git a/vendor/netbox-client/src/netbox-client/models/wireless_lan.cr b/vendor/netbox-client/src/netbox-client/models/wireless_lan.cr
--- a/vendor/netbox-client/src/netbox-client/models/wireless_lan.cr
+++ b/vendor/netbox-client/src/netbox-client/models/wireless_lan.cr
@@ -31,7 +31,7 @@
     property ssid : String
 
     @[JSON::Field(key: "scope", emit_null: false)]
-    property scope : JSON::Any
+    property scope : JSON::Any?
 
     @[JSON::Field(key: "created", emit_null: false)]
     property created : Time
