diff --git a/vendor/netbox-client/src/netbox-client/models/brief_power_panel.cr b/vendor/netbox-client/src/netbox-client/models/brief_power_panel.cr
--- a/vendor/netbox-client/src/netbox-client/models/brief_power_panel.cr
+++ b/vendor/netbox-client/src/netbox-client/models/brief_power_panel.cr
@@ -28,7 +28,7 @@
     property name : String
 
     @[JSON::Field(key: "powerfeed_count", emit_null: false)]
-    property powerfeed_count : Int64
+    property powerfeed_count : Int64?
 
     # Optional properties
     @[JSON::Field(key: "description", emit_null: false)]
